### PR TITLE
Add ffmpeg file dependencies to the MSI

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @metajack 

Generated by using: `depends.exe /oc:out.txt servo.exe`

And then pulling out all the ones in the mingw tree into the MSI.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12720)

### CODE REVIEW

**Comment by larsbergstrom** [comment]
Note that there's supposedly automagical stuff we can do in the future with cmake hooked up to `depends.exe` to automate pulling in all DLL references that aren't satisfied on-system. Though hopefully we won't have *quite* so many once we move to an MSVC toolchain :-)

File: support/windows/Servo.wxs.mako [hunk 1]
```diff
@@ -117,10 +117,154 @@
...
+		    DiskId="1"/>
+	      <File Id="LIBLZMA"
+		    Name="LIBLZMA-5.dll"
+		    Source="c:\msys64\mingw64\bin\LIBLZMA-5.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBMODPLUG"
+		    Name="LIBMODPLUG-1.dll"
+		    Source="c:\msys64\mingw64\bin\LIBMODPLUG-1.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBMP3LAME"
```
Comment (metajack) [inline support/windows/Servo.wxs.mako]: Did mp3 patents expire? Otherwise we can't distribute this.

File: support/windows/Servo.wxs.mako [hunk 2]
```diff
@@ -117,10 +117,154 @@
...
+		    DiskId="1"/>
+	      <File Id="LIBFFI"
+		    Name="LIBFFI-6.dll"
+		    Source="c:\msys64\mingw64\bin\LIBFFI-6.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBGMP"
+		    Name="LIBGMP-10.dll"
+		    Source="c:\msys64\mingw64\bin\LIBGMP-10.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBGNUTLS"
```
Comment (metajack) [inline support/windows/Servo.wxs.mako]: Where is this coming from??

Comment (metajack) [inline support/windows/Servo.wxs.mako]: I'm guessing that ffmpeg has a networking stack and somehow we've linked everything?

**Comment by larsbergstrom** [comment]
All of these deps are coming in recursively from the DLLs that are loaded by those parsers, unfortunately. I can try to play around and see if we can reduce the set of DLLs we ship, but AFAIK, the only real other option is to `cfg` this off on Windows, too (we already did it on Android for similar licensing issues).

**Comment by larsbergstrom** [comment]
@GuillaumeGomez Do you have any ideas here? I'm not sure how to move forward with getting the Windows MSI working due to the presence of new dependencies on libraries we seemingly can't redistribute...

Tonight, I'll do some more tests to find the minimal set of libraries we need. *hopefully* we can avoid issues jack mentioned above.

**Comment by GuillaumeGomez** [comment]
@larsbergstrom: I think the best tip I can give for windows is to look how I made it work with appveyor: https://github.com/GuillaumeGomez/video-metadata-rs/blob/master/appveyor.yml

**Comment by larsbergstrom** [comment]
@GuillaumeGomez oh, that *is* good! The reduced set of files in your win64.zip might be a subset that servo can hopefully run with...

**Comment by GuillaumeGomez** [comment]
@larsbergstrom: It was the hell to test everytime but glad it might be of some use to others!

**Comment by larsbergstrom** [comment]
r? @metajack 

Let's try this one instead. This is from the set that @GuillaumeGomez confirmed works on appveyor.

**Pull request merged**